### ISSUE TITLE
No head tracking on OSX 

### ISSUE THREAD

**Issue by rgoggans** [mabrowning/minecrift#149]
Hi,

I'm trying to get minecrift to work with a DK2 and OSX. The game runs and the display works however there is no head tracking. In the launcher console I have "Error! Couldn't load Oculus: Last initialisation attempt failed". Any ideas?

**Comment by feilen** [comment]
Which version of JRift are you using?

**Comment by StellaArtois** [comment]
Try 0.4.4.3. Although I need to put out a proper Minecrift release soon
anyway with full OS X support.

**Comment by feilen** [comment]
Toss in my latest linux JRift binaries while you're at it!

Were the contribution issues ever resolved?

**Comment by hadfieldn** [comment]
@StellaArtois A new OS X release with head tracking would be terrific. I've  been lurking in the shadows here hoping to get an OS X build working with head tracking (and hopefully eventually contribute to this project), but so far haven't found the time to figure out what I'm doing wrong with my build. 

Playing Minecraft with head tracking is pretty much the reason I got my Rift in the first place. You guys are awesome.

**Comment by StellaArtois** [comment]
Zach, sure. A question though:

- when you built the jrift libs on Linux, did you target a specific platform through Cmake? I have a couple of Ubuntu 14.10 vms where I can build Jrift, but I'm using whatever the default Cmake target is. But slightly worried that means whatever I've built will only run on the very latest Linux distros...

Contribution issues? As in getting it to build without error directly from ./install.sh? It builds for me each time through now, although I need to address the 'Scala on the path' issue on windows at some point.

**Comment by StellaArtois** [comment]
Trying to get a number of fixes into the mod all at once here, but I'll try to get the OSX release out soon! :-)

**Comment by feilen** [comment]
I've built on Ubuntu and Arch. There doesn't seem to be any difference in
libraries between the two that matters to JRift. And neat! I'll try setting
up a prefix again and rolling up things, to try to make more contributions.

**Comment by StellaArtois** [comment]
Ok, cool. Try a clean /install.sh from the Minecrift repo and let me know
if it builds for you. And let me know if JRift 0.4.4.3 also works in your
Linux boxes...


Repo

**Comment by feilen** [comment]
Certainly. If it's the one I built with LTO then it certainly does, as
that's what I've been using for my development. If there's been any changes
I'll be certain to integrate them into my fork.

Speaking of which, you may want to hold off on another release for a month
or so, as it seems like Oculus is gearing up for another PC SDK release.
Although if they lag behind in cross-platform support once again, then
JRift 0.4.4.3 might need bundling.

**Comment by peterfhannon** [comment]
Is head tracking now enabled using DK2 on OSX?

**Comment by StellaArtois** [comment]
The upcoming 1.7.10 Forge release will support it via SDK 0.5. A 1.8 release will follow sometime after that. 

Current releases use an old out-of-date SDK for OS X and Linux that won’t work.

**Comment by feilen** [comment]
The most up to date git version with 0.4.3 still works for me.
